controllers/teleport_master_controller/README

```diff
@@ -0,0 +1,17 @@
+To use this controller, first assign a unique DEF name for each of the robots
+    that you want to control. Also set the supervisor field to TRUE.
+
+Next, add a new Robot Node to the webots world
+(Right click, Add New, Base Nodes, Robot)
+
+Set this new Robot arguments as follows
```

Comment:
It'll probably be easier to just add such a node to the world file, ie add to the end of some world file the following:
```
DEF control Robot {
  controller "teleport_master_controller"
  controllerArgs [
    "control"
  ]
  supervisor TRUE
}
```